README.md

```diff
@@ -7,13 +7,26 @@ the following:
 
 1. Add markdown content to `pages/`
 2. Update `site-config.yml`
-    i. `hub` is the github slug for your active hub. This example defaults to the CDC FluSight hub
+    i. `hub` is the github slug for your active hub. This example defaults to
+       the CDC FluSight hub
```

Comment:
```suggestion
       the CDC FluSight hub (`cdcepi/FluSight-forecast-hub`)
```

Reply:
I still think it's best to explicitly mention we are talking about a repo's owner and name. It's something users will be familiar with while slug is not an intuitive description imho.